locale/en/download/package-manager.md

```diff
@@ -289,6 +290,28 @@ Or install a binary package (if available for your platform) using pkgin:
 pkgin -y install nodejs
 ```
 
+## nvm
+On Unix / OS X systems Node.js built from source can be installed by
+[nvm](https://github.com/creationix/nvm) by installing into the location that nvm expects:
+
+```bash
+$ make install DESTDIR='$(NVM_DIR)/versions/node/' PREFIX='$(FULLVERSION)-pre'
```

Comment:
@ljharb Sorry about this, I was not ignoring you suggestion but failed to get it to work for me. I'm not exactly sure about the cause for this not working yet but from within the Makefile the `nvm_version_path` function cannot be found. I'm not sure if this is because it is a function. I'll take another look tomorrow, but let me know if you have any ideas or suggestions on this. Thanks!